src/main/io/osd.c

```diff
@@ -1447,6 +1459,16 @@ static void osdShowStats(uint16_t endBatteryVoltage)
         osdDisplayStatisticLabel(top++, "MAX SPEED", buff);
     }
 
+    if (osdStatGetState(OSD_STAT_MAX_ESC_TEMP)) {
+        itoa(stats.max_esc_temp, buff, 10);
+        osdDisplayStatisticLabel(top++, "MAX ESC TEMP", buff);
```

Comment:
The appropriate temperature symbol should also be displayed. See `osdGetTemperatureSymbolForSelectedUnit()`.